cmd/operator-sdk/run/packagemanifests/packagemanifests.go

```diff
@@ -35,16 +36,23 @@ func NewCmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "packagemanifests",
-		Short: "Run an Operator organized in the package manifests format with OLM",
+		Short: "Deploy an Operator in the package manifests format with OLM",
+		Long: `'run packagemanifests' deploys an Operator's package manifests with OLM. The command's argument
```

Comment:
if we have a choice between `run packagemanifests` and `run olm`, I'd vote for the latter one.

Reply:
@tengqm the reason we're thinking of using `packagemanifests` is to distinguish it from an upcoming [`run bundle` subcommand](https://github.com/operator-framework/enhancements/blob/master/enhancements/run-bundle.md). The idea is that we want to support various OLM packaging formats for the `run` subcommand, and if we tried to encapsulate all of that under a single `run olm` command, the UX would get unwieldy since there are some differences in how they work.

Reply:
Thanks for the info, @joelanford. I am just not a big fan of long commands, :).

Reply:
@estroz Perhaps we could add a cobra alias for this (e.g. `operator-sdk run pm`)?

Reply:
SGTM, I'll make a follow-up.

Reply:
https://github.com/operator-framework/operator-sdk/pull/3314